DatabaseController.js

```diff
@@ -0,0 +1,56 @@
+import mysql from 'mysql';
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const DatabaseController = () => {
+    const db = mysql.createConnection({
+        host: process.env.HOST,
+        user: process.env.USER,
+        password: process.env.PASSWORD,
+        database: process.env.DATABASE
+    });
+
+    db.connect(function (err) {
+        if (err) {
+            throw err;
+        }
+    });
+
+    const insert = ({table, data = []}) => {
+        let values = '';
```

Comment:
Add an early return here in case data is an empty array as there's nothing to insert in that case.